### ISSUE TITLE
AdAway block web after whitelist

### ISSUE THREAD

**Issue by oblus** [AdAway/AdAway#3342]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

After Ive put in whitelist uploadrar.com and *.uploadrar.com AdAway still blocking this web site.

### Expected Behavior

To not block uploadrar.com while his entry is written in whitelist

### Steps To Reproduce

You know how to add to whitelist - maybe I doing smth wrong?

### Device Configuration

- AdAway version:  newest
- Ad-blocker mode: root
- Device: s10e
- Android version: 10
- Root version:  magisk 24306

**Comment by LtPatterson** [comment]
I'm seeing the same thing on *avantlink.com* wildcard whitelisting, it appears to still be blocking.

**Comment by Zipfer** [comment]
Have the same issue. Pixel 5a. Android 13.
The allowed list doesn't work. The pause of the service doesn't work. The pause of the service doesn't unlock everything (I can see that /etc/hosts is empty)

**Comment by LtPatterson** [comment]
> Have the same issue. Pixel 5a. Android 13. The allowed list doesn't work. The pause of the service doesn't work. The pause of the service doesn't unlock everything (I can see that /etc/hosts is empty)

check your browser settings if using brave or kiwi, may be a duplicate list issue. update filters, this seems to be hit and miss for me on some.

**Comment by Zipfer** [comment]
> > Have the same issue. Pixel 5a. Android 13. The allowed list doesn't work. The pause of the service doesn't work. The pause of the service doesn't unlock everything (I can see that /etc/hosts is empty)
> 
> check your browser settings if using brave or kiwi, may be a duplicate list issue. update filters, this seems to be hit and miss for me on some.
No, it's not a dupe blocker list. I can see the Adaway icon on the page background instead of the requested page.

**Comment by oblus** [comment]
I think Pausing only work if you reOpen browser (I think browser stores hosts in memory )... 
but this still doesnt fix WhiteList!

**Comment by Zipfer** [comment]
Nope, I've tried, to pause the AdAway, stop the browser, and reopen the same icon.
Allowed doesn't help, reboot after adding sites to Allowed doesn't help.
Amazon app doesn't work, Chase app doesn't work because of their analytics.* subdomain is blocked
All these apps work only the first 10 seconds after reboot, then popup of Adaway root granted permissions and no Chase, no Amazon main page
